### PR TITLE
Misc gateway updates

### PR DESCRIPTION
**Pull request by connorckong**
### Description

What did you change and why?
This PR contains a handful of minor changes, including tweaked UTMs, updates to the getting started guide to reference Kong Manager, and a CTA to try Konnect

Include any supporting resources, e.g. link to a Jira ticket, GH issue, FTI, Slack, Aha, etc.

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/gateway/index.md [hunk 1]
```diff
@@ -82,21 +82,40 @@ the [PDK reference](/gateway/{{page.kong_version}}/plugin-development/pdk/), and
 
 ## Packages and modes
 
-{{site.base_gateway}} is available in the following modes:
+There are two ways to deploy {{site.base_gateway}} - {{ site.konnect_saas }} cloud and on-premises. If you're trying
```
Comment (acgoldsmith) [inline app/_src/gateway/index.md:85]:
```suggestion
There are two ways to deploy {{site.base_gateway}}: cloud-hosted on {{ site.konnect_saas }}, and on-premises. If you're trying
```

File: app/_src/gateway/index.md [hunk 2]
```diff
@@ -82,21 +82,40 @@ the [PDK reference](/gateway/{{page.kong_version}}/plugin-development/pdk/), and
...
+{{site.base_gateway}} features.
+See the [{{site.konnect_saas}} documentation](/konnect/) and the
+**Plus**-labelled plugins on the [Plugin Hub](/hub/#plus) for more information.
+* <span class="badge enterprise"></span> With an **Enterprise** subscription,
+  it also includes:
+    * [Dev Portal](/konnect/dev-portal/)
+    * [Analytics](/konnect/analytics/)
+    * [RBAC](/gateway/{{page.kong_version}}/admin-api/rbac/reference/)
+    * [Enterprise plugins](/hub/#ee-compat)
+
```
Comment (acgoldsmith) [inline app/_src/gateway/index.md:105]:
```suggestion
out {{site.base_gateway}} for the first time we, recommend starting with [{{ site.konnect_saas }}](https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=gateway-konnect&utm_content=gateway-mode-overview).

###Cloud-hosted with Konnect

**{{site.base_gateway}}** (available in
[Free, Plus, or Enterprise modes](https://konghq.com/pricing)): Kong's API gateway
with added functionality.
* <span class="badge free"></span> In **Free mode**,
  this package gives you access to Konnect to help configure your services, routes and plugins.
* <span class="badge plus"></span> In **Plus mode**, you have access to more
{{site.base_gateway}} features.
See the [Konnect documentation](/konnect/) and the
**Plus**-labelled plugins on the [Plugin Hub](/hub/#plus) for more information.
* <span class="badge enterprise"></span> With an **Enterprise** subscription,
  it also includes:
    * [Dev Portal](/konnect/dev-portal/)
    * [Analytics](/konnect/analytics/)
    * [RBAC](/gateway/{{page.kong_version}}/admin-api/rbac/reference/)
    * [Enterprise plugins](/hub/#ee-compat)

```

Comment (acgoldsmith) [reply app/_src/gateway/index.md:105]: Biggest change here is current product naming conventions are Kong Konnect and then just Konnect... I'm retaining the cloud messaging part but removing from product name.

File: app/_src/gateway/install/docker/index.md [hunk 3]
```diff
@@ -22,6 +22,10 @@ The {{site.base_gateway}} software is governed by the
 
 ## Prerequisites
 
+{:.note}
+> **Note:**
+> If you want to run Kong Gateway without managing a control plane or a database, [you can get started with {{ site.konnect_saas }}](https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=gateway-konnect&utm_content=docker-install) in under 5 minutes using our Docker quick start script.
```
Comment (acgoldsmith) [inline app/_src/gateway/install/docker/index.md:27]:
```suggestion
> If you want to run Kong Gateway without managing a control plane or a database, [you can get started with Konnect](https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=gateway-konnect&utm_content=docker-install) in under 5 minutes using our Docker quick start script.
```

**Comment by acgoldsmith** [comment]
reviewing changes w/ @connorckong prior to merging.

**Comment by connorckong** [comment]
Your suggestion/commits seem good to me, @acgoldsmith - are we able to merge?

**Review by acgoldsmith — APPROVED** [review]
LGTM

**Pull request merged**